tests/foreman/ui/test_gpgkey.py

```diff
@@ -671,23 +603,19 @@ def test_info_key_1(self):
 
         """
 
-        pass
 
-
-@ddt
 class GPGKeyProductAssociate(UITestCase):
```

Comment:
Note: All the associations test were data driven previously.  There is no need for the associations to be tested for each gpgkey name variation.  All these data driven tests are replaced by single data point tests which is achieved by using the random gpgkey name creator get_random_gpgkey_name().  This returns one datapoint from generate_strings_list().

Advantages:
* All different gpgkey name variations will be tested across 20 different variation tests by using get_random_gpgkey_name().
* Before change - Since there str twenty associations tests - this amounts to 20 * 7 = 140 UI tests which would run forever.  Now with the new changes, the number of tests is 20 * 1 = 20 which is lot faster and also gives needed sufficient coverage.